### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by blowmage**
This file should be added to the root directory of gcloud projects.
Follow AngularJS's lead by adapting the Contributor Covenant.

### CODE REVIEW

**Comment by quartzmo** [comment]
Other documents follow a convention of `<root_dir>/<topic_name>/readme.md`. For example: `troubleshooting/readme.md`. I think this may have to do with the Angular application used to dynamically include content, although I'm not absolutely certain of the origin of this convention.

**Comment by stephenplusplus** [comment]
I only did that to group each logically, thinking they might each have their own images or related documents.

**Comment by blowmage** [comment]
The convention I've seen on other projects is that the file is in the root of the repo, similar to README. This is where I believe folks will look for and expect to see it.

**Comment by stephenplusplus** [comment]
Makes sense to me. Do we need both formats?

**Comment by blowmage** [comment]
I added both formats here for completeness, but each gcloud project should pick the format that makes the most sense. If you aren't already using markdown then you can choose the plain text format.

**Comment by stephenplusplus** [comment]
I'd rather just have the markdown file, since the rest of our repo is just markdown. Each gcloud project is GH hosted (I think?), so markdown should be ok.

**Comment by blowmage** [comment]
Removed plain text version.

**Comment by stephenplusplus** [comment]
Cool, thanks. I suppose leave it up to @jgeewax to merge?

**Comment by blowmage** [comment]
Once this PR is accepted I'll send PRs to all the gcloud projects to add the file.

File: conduct/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
```
Comment (jgeewax) [inline conduct/CODE_OF_CONDUCT.md]: Any chance we can follow the semantic line break style? http://rhodesmill.org/brandon/2012/one-sentence-per-line/

**Comment by jgeewax** [comment]
Content :+1:

**Comment by blowmage** [comment]
How about wrapping at 80 characters?

**Comment by jgeewax** [comment]
80-characters kind of misses the point of the semantic breaks... The goal is to have a break be "at the end of a clause" (code already has this -- for the most part).

**Comment by blowmage** [comment]
Can someone take a stab at adding semantic breaks to this document. My attempts so far have been uneven and I feel have hurt the readability. /cc @quartzmo

**Pull request merged**